### DISCUSSION TITLE
Analyze PDFs and Documents

### DISCUSSION THREAD

**Discussion by kalle07** [oobabooga/text-generation-webui#5099] [Ideas]
hey,

superboogaV2 only works with TXT files why not with PDF, it give already open source code in privateGPT (https://github.com/imartinez/privateGPT) , maybe only copy&past   ;)

**Comment by TomLucidor** [comment]
Theow in Khoj and GPT4All and how they can handle LogSeq and Obsidian
See also https://github.com/oobabooga/text-generation-webui/discussions/1933
